vmcheck.py

```diff
@@ -0,0 +1,379 @@
+                                'type': 'Driver',
+                                'indicator': dll_name,
+                                'description': 'VM/Sandbox driver detected',
+                                'severity': 'Critical',
+                                'info': f'Process: {proc_name} (PID: {proc_pid}) | Path: {dll_path}'
+                            })
+                    except Exception:
+                        continue
+            except Exception:
+                continue
+        
+        return score, detections
+    
+    def _calculate_verdict(self, total_score: int, detection_count: int) -> Tuple[str, str]:
+        """Calculate final verdict based on score and detections."""
+        if total_score >= 50 or detection_count >= 5:
+            return "SANDBOX", "High confidence - Multiple sandbox indicators detected"
+        elif total_score >= 30 or detection_count >= 3:
+            return "LIKELY SANDBOX", "Medium confidence - Several suspicious indicators"
+        elif total_score >= 15 or detection_count >= 1:
+            return "SUSPICIOUS", "Low confidence - Few indicators detected"
+        else:
+            return "REAL MACHINE", "No significant sandbox indicators found"
+    
+    def _generator(self):
+        kernel_module_name = self.config["kernel"]
+        verbose = self.config.get("verbose", False)
+        
+        all_detections = []
+        total_score = 0
+        
+        # Run all checks
+        vollog.info("Checking for sandbox processes...")
+        proc_score, proc_detections = self._check_processes(kernel_module_name)
+        all_detections.extend(proc_detections)
+        total_score += proc_score
+        
+        vollog.info("Checking for sandbox DLLs...")
+        dll_score, dll_detections = self._check_dlls(kernel_module_name)
+        all_detections.extend(dll_detections)
+        total_score += dll_score
+        
+        vollog.info("Checking system information...")
+        sys_score, sys_detections = self._check_system_info(kernel_module_name)
+        all_detections.extend(sys_detections)
+        total_score += sys_score
+        
+        vollog.info("Checking for VM drivers...")
+        drv_score, drv_detections = self._check_drivers(kernel_module_name)
+        all_detections.extend(drv_detections)
+        total_score += drv_score
+        
+        # Calculate verdict
+        verdict, confidence = self._calculate_verdict(total_score, len(all_detections))
+        
+        # Yield summary
+        yield (0, (
+            "SUMMARY",
+            verdict,
+            f"Score: {total_score} | Detections: {len(all_detections)}",
+            confidence,
+            ""
+        ))
+        
+        yield (0, ("", "", "", "", ""))
+        
+        # Yield detailed detections if verbose or if detections found
+        if verbose or all_detections:
+            for detection in all_detections:
+                yield (0, (

```

Comment:
You can yield these with a 1, to indent them in a tree view, instead of 0 here, if you wish to differentiate them.